File: core/src/main/java/vn/loitp/function/epub/model/BookInfoData.java
package vn.loitp.function.epub.model;

public class BookInfoData {
    private static final BookInfoData ourInstance = new BookInfoData();

    public static BookInfoData getInstance() {
        return ourInstance;
    }

    private BookInfoData() {
    }

    private BookInfo bookInfo;

    public BookInfo getBookInfo() {
        return bookInfo;
    }

    public void setBookInfo(BookInfo bookInfo) {
        this.bookInfo = bookInfo;
    }
}

File: core/src/main/java/com/loitpcore/function/epub/model/BookInfoData.kt
package com.loitpcore.function.epub.model

import androidx.annotation.Keep
import com.loitpcore.core.base.BaseModel

@Keep
class BookInfoData private constructor() : BaseModel() {

    companion object {
        val instance = BookInfoData()
    }

    var bookInfo: BookInfo? = null
}
